### ISSUE TITLE
Portal 2 Bug: Credits videos do not play

### ISSUE THREAD

**Issue by Equalizer5118** [StrataSource/Portal-2-Community-Edition#1575]
### Describe the bug

After the final boss fight, there is supposed to be an aperture screen that shows the credits, and afterwards a movie of wheatley apologizing. These do not play, only the song lyrics play.

This does NOT happen when loading directly into `sp_a5_credits`, only after beating wheatley in `sp_a4_finale4`
![20231104112930_1](https://github.com/StrataSource/Portal-2-Community-Edition/assets/113145248/b785024e-cd51-4659-ae77-0634c8c9c406)
![20231104112932_1](https://github.com/StrataSource/Portal-2-Community-Edition/assets/113145248/36c9f3e3-4b8c-44a1-9d7b-7329dbe60368)
![20231104113156_1](https://github.com/StrataSource/Portal-2-Community-Edition/assets/113145248/f61fcf07-882f-40d6-ab52-2e9fad561bc5)

(Notice the caption of space core saying he's in space. There is supposed to be that ending movie here. )

### To Reproduce

1. Open `sp_a4_finale4`
2. Beat wheatley
3. Watch all the cutscenes
4. Observe bug

### Operating System

Tested on Windows 11

**Comment by Luke18033** [comment]
This is likely caused by StrataSource/Engine#971

**Comment by ertup0** [comment]
This seems to work on the latest version

**Comment by lenship2** [comment]
Superseded by #1656
